fedgov-cv-ISDcreator-webapp/src/main/webapp/js/mapping.js

```diff
@@ -2543,8 +2543,8 @@ function onRoadAuthorityIdTypeChangeCallback(roadAuthorityIdType){
 	if(roadAuthorityIdType!==""){
 		$("#road_authority_id").attr('data-parsley-required', true);
 		if (roadAuthorityIdType === "full") {
-            roadAuthorityIdInput.attr("data-parsley-pattern", "([0-2](\.[0-9]+)+)");
-            roadAuthorityIdInput.attr("data-parsley-pattern-message", "For Full RAID, the first integer must be 0-2, with at least one period-separated integer.");
+            roadAuthorityIdInput.attr("data-parsley-pattern", "^(([01]\.(0|[1-9]|[1-2][0-9]|3[0-9]))|(2\.[0-9]+))(\.[0-9]+)*$");
+            roadAuthorityIdInput.attr("data-parsley-pattern-message", "For Full RAID, the first integer must be 0-2, with at least one period-separated integer. If the first integer is either 0 or 1, the second integer cannot exceed 40.");
```

Comment:
Could you make a slight change to the language here to make it clearer for the user:

"If the first integer is either 0 or 1, the second integer must be between 0-39."